src/routes/resources/resources.handlers.ts

```
@@ -52,25 +52,44 @@ export const getAll: AppRouteHandler<GetAllRoute> = async (c) => {
   const categoryName = c.req.query("category") ?? "";
   const tagsParam = c.req.query("tags") ?? "";
 
+  const cursor = c.req.query("cursor") ?? undefined;
+  const limitParam = c.req.query("limit") ?? "10";
+  const limit = parseInt(limitParam, 10);
```

ESLint

[error] Prefer `Number.parseInt` over `parseInt`.

(unicorn/prefer-number-properties)